### DISCUSSION TITLE
How to change language of DatePicker / TimePicker / CalendarDatePicker

### DISCUSSION THREAD

**Discussion by boexler** [microsoft/microsoft-ui-xaml#7699] [Q&A]
I'm just wondering how I can change the language of the `DatePicker` / `TimePicker` / `CalendarDatePicker`.
The language of my application is independent of my operating system.
The user can change the language live in the application and all text fields are automatically translated (I developed my own system for this and do not use any Microsoft functions).

Now I ask myself how I can change both the language and the time format.

preview from WinUI3 Gallery
![grafik](https://user-images.githubusercontent.com/17551597/187408793-01845981-214c-49d3-a1b8-59aee1482fa1.png)
![grafik](https://user-images.githubusercontent.com/17551597/187408931-84bc07f8-8209-49a1-bbdf-1f4348ec914d.png)

preview from my application
![grafik](https://user-images.githubusercontent.com/17551597/187408865-dd71e557-ea0c-4b90-8782-e7a02e919a8d.png)

I'm using **WinUI 3**

Many Thanks

**Comment by AndrewKeepCoding** [comment]
I guess you can't (at least for the moment). But you might be able to achieve what you want like I did in this [video](https://youtu.be/ai-koyvgbWY). I'm planning on make changes on the ``DatePicker`` control. The month selector is too wide for numbers instead of the names. Hope this helps.😎
